docs/content/latest/reference/configuration/yugabyted.md

```diff
@@ -169,6 +169,14 @@ Enable or disable the "call home" feature that sends analytics data to Yugabyte.
 
 Enable or disable the webserver UI. Default is `false`.
 
+##### --master_flags *master_flags*
```

Comment:
Might be useful to provide an external link to all the flag options that are potentially available to the user. Maybe this: https://docs.yugabyte.com/latest/reference/configuration/yb-master/.

Reply:
linked here https://github.com/yugabyte/yugabyte-db/pull/6689/commits/8381b5ed187431a4804521fc2850c01a2a4e1a65